src/nl/hannahsten/texifyidea/util/Commands.kt

```diff
@@ -38,6 +45,90 @@ fun getIncludeCommands(): Set<String> {
         .toSet()
 }
 
+/**
+ * Inserts a usepackage statement for the given package in a certain file.
+ *
+ * @param file
+ *          The file to add the usepackage statement to.
+ * @param packageName
+ *          The name of the package to insert.
+ * @param parameters
+ *          Parameters to add to the statement, `null` or empty string for no parameters.
+ */
+fun insertCommandDefinition(file: PsiFile, commandText: String, newCommandName: String = "mycommand"): PsiElement? {
```

Comment:
I know they are very similar. I copied one to make the other. I wasnt sure how similar they would end up being, so I figured I would remerge them later. Im still not sure I have done it right because newlines get really messy

Reply:
I can have a look to see if I can refactor it

Reply:
totally forgot about this. I should be able to fix it within 24 hrs

Reply:
I looked into it, and there are differences in each block of the code, the way that the insert location is chosen does not track the `prependNewLine` variable, `command` is constructed differently, the insert `runWriteAction` also has differences in order to get the whitespacing right, and the commands that are allowed are all different. They definitely smell the same, but the taste just isnt right.

Reply:
okay, thanks for checking. I'll have a look to see if I want to change anything or not